Debian build failes because missing dependency: python3-yaml

### Describe the bug

Debian package can not be built because missing dependency python3-yaml.

### Steps to reproduce

1. git clone ..
2. try to build package on debian (12): dpkg-buildpackage -rfakeroot
3. see error about missing yaml python module

### Expected behavior

Build successfully

### Build version

````
git main
````

### Operating system

Debian 12

### Build log

<!--- Paste (partial) output from `/var/log/cobbler/cobbler.log` -->
````
I: pybuild base:240: cd /root/cobbler/.pybuild/cpython3_3.11_cobbler/build; python3.11 -m unittest discover -v 
cobbler.actions.buildiso (unittest.loader._FailedTest.cobbler.actions.buildiso) ... ERROR
cobbler.modules.managers (unittest.loader._FailedTest.cobbler.modules.managers) ... ERROR
cobbler.settings (unittest.loader._FailedTest.cobbler.settings) ... ERROR
cobbler.utils (unittest.loader._FailedTest.cobbler.utils) ... ERROR
tests.performance (unittest.loader._FailedTest.tests.performance) ... ERROR
...
ModuleNotFoundError: No module named 'yaml'


======================================================================
ERROR: cobbler.modules.managers (unittest.loader._FailedTest.cobbler.modules.managers)
----------------------------------------------------------------------
...
ModuleNotFoundError: No module named 'yaml'


======================================================================
ERROR: cobbler.settings (unittest.loader._FailedTest.cobbler.settings)
----------------------------------------------------------------------
...
ModuleNotFoundError: No module named 'yaml'


======================================================================
ERROR: cobbler.utils (unittest.loader._FailedTest.cobbler.utils)
----------------------------------------------------------------------
...
ModuleNotFoundError: No module named 'yaml'


======================================================================
ERROR: tests.performance (unittest.loader._FailedTest.tests.performance)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests.performance
Traceback (most recent call last):
  File "/usr/lib/python3.11/unittest/loader.py", line 440, in _find_test_path
    package = self._get_module_from_name(name)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/unittest/loader.py", line 350, in _get_module_from_name
    __import__(name)
  File "/root/cobbler/.pybuild/cpython3_3.11_cobbler/build/tests/performance/__init__.py", line 8, in <module>
    from cobbler.api import CobblerAPI
  File "/root/cobbler/.pybuild/cpython3_3.11_cobbler/build/cobbler/api.py", line 128, in <module>
    from cobbler import (
  File "/root/cobbler/.pybuild/cpython3_3.11_cobbler/build/cobbler/autoinstall_manager.py", line 10, in <module>
    from cobbler import autoinstallgen, utils
  File "/root/cobbler/.pybuild/cpython3_3.11_cobbler/build/cobbler/autoinstallgen.py", line 13, in <module>
    from cobbler import templar, utils, validate
  File "/root/cobbler/.pybuild/cpython3_3.11_cobbler/build/cobbler/templar.py", line 17, in <module>
    from cobbler import utils
  File "/root/cobbler/.pybuild/cpython3_3.11_cobbler/build/cobbler/utils/__init__.py", line 40, in <module>
    from cobbler import enums, settings
  File "/root/cobbler/.pybuild/cpython3_3.11_cobbler/build/cobbler/settings/__init__.py", line 22, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'


----------------------------------------------------------------------
Ran 5 tests in 0.001s

FAILED (errors=5)
E: pybuild pybuild:388: test: plugin distutils failed with: exit code=1: cd /root/cobbler/.pybuild/cpython3_3.11_cobbler/build; python3.11 -m unittest discover -v 
dh_auto_test: error: pybuild --test -i python{version} -p 3.11 returned exit code 13
make: *** [debian/rules:15: build] Error 13
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2

````

--- a/changelog.d/3668.fixed
+++ b/changelog.d/3668.fixed
@@ -1,0 +1,1 @@
+Added python3-yaml to debian build-dependencies to fix unittests.

--- a/debian/control
+++ b/debian/control
@@ -13,6 +13,7 @@ Build-Depends:
  python3-cheetah,
  python3-dns,
  python3-sphinx,
+ python3-yaml,
  dh-python,
  debhelper (>=12)
 Standards-Version: 4.5.1

